bellatrix.web/src/main/java/solutions/bellatrix/web/components/WebComponent.java

```diff
@@ -148,6 +157,10 @@ public String getTitle() {
         return getAttribute("title");
     }
 
+    public WebComponent getParent() {
```

Comment:
Sounds ambiguous with parentWrappedElement field and its get method, immediate difference being only the type